docs/advanced_usage.rst

```diff
@@ -0,0 +1,333 @@
+Advanced Usage
+==============
+
+Agent Configuration
+-------------------
+
+If the Datadog Agent is on a separate host from your application, you can modify
+the default ``ddtrace.tracer`` object to utilize another hostname and port. Here
+is a small example showcasing this::
+
+    from ddtrace import tracer
+
+    tracer.configure(hostname=<YOUR_HOST>, port=<YOUR_PORT>)
+
+By default, these will be set to localhost and 8126 respectively.
+
+Distributed Tracing
```

Comment:
We should add a reference to our official shared documentation describing what distributed tracing is.

Reply:
I think we can keep it in place for now as we'll keep this documentation more technical and leave the corp-site docs more general and high-level.